src/playbacks/flashls/flashls.js

```diff
@@ -675,6 +675,10 @@ export default class FlasHLS extends BaseFlashPlayback {
     }
   }
 
+  get dvrEnabled() {
+    return this._dvrEnabled
```

Comment:
I think `!!this._dvrEnabled` would be better to ensure it's always a boolean, as this starts as `undefined`.